content/en/docs/reference/config/meta/v1beta1/istio-status/index.html

```diff
@@ -0,0 +1,191 @@
+---
+WARNING: THIS IS AN AUTO-GENERATED FILE, DO NOT EDIT. PLEASE MODIFY THE ORIGINAL SOURCE IN THE 'https://github.com/istio/api' REPO
+source_repo: https://github.com/istio/api
+title: Istio Status
+description: Common status field for all istio collections.
+location: https://istio.io/docs/reference/config/meta/v1beta1/istio-status.html
+layout: protoc-gen-docs
+generator: protoc-gen-docs
+hide_from_docs
```

Comment:
@therealmitchconnors We need to remove this line as it fails linting. Are you expecting this file to not be placed in the docs? If so, the way to do that is to remove the location: tag.

Reply:
Should be fixed by https://github.com/istio/api/pull/1524 when it's merged.